plugins/services/src/js/reducers/serviceForm/HealthChecks.js

```diff
@@ -13,7 +13,7 @@ function mapHealthChecks(item) {
     newItem.protocol = item.protocol;
     if (item.protocol.toUpperCase() === 'COMMAND' && item.command != null) {
       newItem.command = {
-        command: item.command
```

Comment:
we gotta update this in the form too, so the HealthCheck partial looks for errors in `command.command`